knative-operator/pkg/common/constants.go

```diff
@@ -6,4 +6,6 @@ const (
 	ServingOwnerNamespace            = "serving.knative.openshift.io/ownerNamespace"
 	ServerlessOperatorOwnerName      = "operator.knative.openshift.io/ownerName"
 	ServerlessOperatorOwnerNamespace = "operator.knative.openshift.io/ownerNamespace"
+	KafkaOwnerName                   = "knativekafkas.operator.serverless.openshift.io/ownerName"
+	KafkaOwnerNamespace              = "knativekafkas.operator.serverless.openshift.io/ownerNamespace"
```

Comment:
Thinking about this, this is actually kinda annoying isn't it?

Maybe we should normalize all of them and add a third one that denotes the owner type so can distinguish in the respective operators.

For this specifically that could mean we do these:

```
operator.serverless.openshift.io/ownerName: "foo"
operator.serverless.openshift.io/ownerNamespace: "knative-eventing"
operator.serverless.openshift.io/ownerType: "KnativeKafka"
```

Reply:
Do we move all artifacts than to `knative-eventing` ?

Reply:
Nah, that was only an example to make clear what I think this could look like. The namespace would of course be dynamic to the resource.

Reply:
I can do this for KnativeKafka controller only.

For consistency, I would like to change others too, but I am worried a bit about what happens during an upgrade

Reply:
Yeah we have to do the changes to the others graduall (add them in one release, only start depending on them in the next one).

Reply:
Let me try doing this later ok? Not sure how this would work because we don't have the CR instance at the creation time of the watches

Reply:
Yes, let's just make sure we do this before we enable this controller. I can send a patch with how I think it should look like too,  you don't have to do everything :).

Reply:
Cool, I will ping you just before I send a PR to enable the controller.